src/status_im2/contexts/quo_preview/transaction_sheet/transaction_sheet.cljs

```diff
@@ -0,0 +1,91 @@
+(ns status-im2.contexts.quo-preview.transaction-sheet.transaction-sheet
+  (:require [quo2.components.tabs.account-selector :as acc-sel]
+            [utils.i18n :as i18n]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [status-im2.contexts.quo-preview.tabs.segmented-tab]
+            [quo2.components.tabs.segmented-tab :as quo2]
+            [status-im2.contexts.shell.jump-to.utils :as utils]
+            [quo2.components.inputs.locked-input.view :as locked-input]
+            [quo2.components.markdown.text :as text]))
+
+(let [{:keys [width]} (utils/dimensions)]
+  (def screen-width width))
+(def account-selector-width (* 0.895 screen-width))
+(def shared-selector-list-data
+  {:show-label?  false
+   :transparent? false
+   :style        {:width  account-selector-width
+                  :height 40}})
+(def unique-selector-list-data
+  [{:account-text  "Drakaris account"
+    :account-emoji "🔥"}
+   {:account-text  "Daenerys account"
+    :account-emoji "👸"}])
+(def selector-list-data
+  (map #(merge % shared-selector-list-data)
+       unique-selector-list-data))
+
+(def label-style
+  {:color          colors/neutral-50
+   :letter-spacing -0.039
+   :line-height    18.2
+   :font-weight    :normal
+   :font-size      13})
+
+(defn- render-account-selectors
+  [item]
+  [rn/view {:style {:margin-right 10}}
+   [rn/touchable-opacity {:on-press #(js/alert (str "Pressed " (item :account-text)))}
+    [acc-sel/account-selector item]]])
+(defn preview-transaction-sheet
```

Comment:
@J-Son89 Is putting together the transaction sheet in the preview the way to go? or should we create a separate transaction-sheet component?

Reply:
@tumanov-alex is an interview candidate so we asked him to put this on the preview so he can demonstrate his abilities. It's a combination of a few components which we can follow up to put in their own preview screens. So all good here 👍